packages/launchpad/src/error/BaseError.vue

```diff
@@ -9,6 +9,8 @@
           {{ headerText }}
         </slot>
       </h1>
+      <!-- eslint-disable vue/multiline-html-element-content-newline  -->
```

Comment:
Why do we need these? If the rule in impractical, should we turn it off?

Reply:
The problem we're avoiding here is extra whitespace added by the line breaks this rule requires. It _sometimes_ leads to weird extra space being underlined, if there's another inline element next to a link. I'd support turning this rule off completely, since the internal whitespace between tags can be significant in ways that aren't always obvious right off the bat.